src/Components/App/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
```

Comment:
Please delete all this starter stuff and replace it with a simple p saying component rendering for now.